packages/suite-desktop/src-electron/modules/auto-updater.ts

```diff
@@ -2,13 +2,17 @@
  * Auto Updater feature (notify, download, install)
  */
 
-import { app, ipcMain, BrowserWindow } from 'electron';
+import { app, ipcMain } from 'electron';
 import { autoUpdater, CancellationToken } from 'electron-updater';
+import { b2t } from '@lib/utils';
+import { toHumanReadable } from '@suite/utils/suite/file';
```

Comment:
```suggestion
import { toHumanReadable } from '@suite-utils/file';
```

Reply:
The TS Config for `src-electron` is not as extensive as the ones in the rest of the project. The config itself needed a few tweaks and originally didn't include paths/aliases to other parts of the project, so I just added `@suite` to point to `packages/suite/src`. That's why I have the "ugly" paths in there.